crossplane.yaml

```diff
@@ -43,4 +43,6 @@ spec:
     version: ">=v0.14.0-0"
   dependsOn:
     - provider: crossplane/provider-aws
-      version: ">=v0.14.0-0"
+      version: ">=v0.15.0-0"
```

Comment:
Are there specific things we want to avoid in the existing versions here? If not, since we use `>=` already the newest versions of these dependencies will be used anyway.

Reply:
fair point, keeping at 0.14.0

Reply:
@prasek just noting here that we may end up having to bump these in the future if https://github.com/crossplane/crossplane/pull/2015#discussion_r539747551 happens, but we are good for now :+1:

Reply:
We may want to make these deps on `registry.upbound.io/crossplane/provider-aws`, etc.

Reply:
done!